.conda/README.md

```diff
@@ -1,30 +1,32 @@
 # Publish to conda
 
-There is two systems to publish to conda:
-- A fully automatic CI that publish to an _openfisca_ channel. See below for more information.
-- A more complex in Conda-Forge CI, that publish to _Conda-Forge_ channel. See https://www.youtube.com/watch?v=N2XwK9BkJpA for an introduction to Conda-Forge. We do not use it for this project.
+There are two publishing systems for `openfisca-survey-manager` conda packages:

```

Comment:
If we do not use Conda-Forge, there is only one system for `openfisca-survey-manager`, right ?

Reply:
Yes, you are right, I introduced an error when I updated the sentence. Sorry, fixed!